draft-ietf-quic-recovery.md

```diff
@@ -812,17 +785,11 @@ Pseudocode for SetLossDetectionTimer follows:
       loss_detection_timer.set(loss_time)
       return
 
-    // RTO or TLP timer
-    // Calculate RTO duration
+    // PTO timer. Calculate PTO duration
```

Comment:
Nit: I feel like having either `PTO timer` or `Calculate PTO duration` would be cleaner. Having both seems a bit overdone.